blindsign/blindrsa/brsa.go

```diff
@@ -45,6 +49,14 @@ type DeterminsiticRSAVerifier struct {
 	hash hash.Hash
 }
 
+type BRSAVerifier interface {
```

Comment:
Blind is already in the name of the package, twice. I don't think we need to repeat "B" for blind. (Just like it's `big.Int` and not `big.BigInt`.)